portal/v2/src/RegionList.tsx

```diff
@@ -0,0 +1,53 @@
+import { Component } from "react"
+import { Nav, INavLinkGroup, INavStyles, IRenderGroupHeaderProps, IRenderFunction, Stack, Text } from "@fluentui/react"
+
+const navStyles: Partial<INavStyles> = {
+  chevronIcon: {
+    visibility: 'hidden'
+  }
+};
+
+interface RegionComponentProps {
+  item: IRegionList
+}
+
+interface IRegionComponentState {
+  item: IRegion // why both state and props?
```

Comment:
nit: why indeed? :stuck_out_tongue:

Reply:
I was under the impression props were for passing in values into a it class and state was for maintaining the values of the class during runtime. If this is the incorrect way to this I am happy to change it

Reply:
I don't know that it is incorrect, I was just curious the answer to `why`. Thank you!